pkg/webhooks/jobset_webhook_test.go

```diff
@@ -1492,6 +1492,13 @@ func TestValidateUpdate(t *testing.T) {
 										ObjectMeta: metav1.ObjectMeta{
 											Annotations: map[string]string{"key": "value"},
 										},
+										Spec: corev1.PodSpec{
```

Comment:
Can you add a new test case for scheduling gates?

Reply:
Sure, I pushed the change. PTAL